src/Keboola/GoogleSheetsWriter/Sheet.php

```diff
@@ -226,12 +218,12 @@ private function findSheetPropertiesById(array $sheets, int $sheetId) : array
         return array_shift($results);
     }
 
-    public function getRange(string $sheetTitle, int $columnCount, int $rowOffset = 1, int $rowLimit = 1000) : string
+    public function getRange(string $sheetTitle, int $columnCount, int $rowStart = 1, int $rowEnd = 1000) : string
     {
         $lastColumn = $this->columnToLetter($columnCount);
 
-        $start = 'A' . $rowOffset;
-        $end = $lastColumn . ($rowOffset + $rowLimit - 1);
+        $start = 'A' . $rowStart;
+        $end = $lastColumn . ($rowStart + $rowEnd - 1);
```

Comment:
tohle mi nepřijde správně - např. $rowStart = 20, $rowEnd = 100 - takže $end bude 20+100-1 (tj. 119)? Proč? Ledaže by $rowEnd bylo spíš $rowCount.

Reply:
No ja som to prave premenoval, ale asi mas pravdu, ze to teraz nedava zmysel :) Ale funguje to spravne, je tam na to test ;)